src/Humanizer.Tests/Localisation/fi-FI/NumberToWordsTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Xunit;
+using Xunit.Extensions;
+
+namespace Humanizer.Tests.Localisation.fiFI
+{
+    public class NumberToWordsTests : AmbientCulture
+    {
+        public NumberToWordsTests() : base("fi-Fi") { }
+
+        [Theory]
+        [InlineData(0, "nolla")]
+        [InlineData(1, "yksi")]
+        [InlineData(11, "yksitoista")]
+        [InlineData(15, "viisitoista")]
+        [InlineData(19, "yhdeksäntoista")]
+        [InlineData(20, "kaksikymmentä")]
+        [InlineData(25, "kaksikymmentäviisi")]
+        [InlineData(50, "viisikymmentä")]
+        [InlineData(90, "yhdeksänkymmentä")]
+        [InlineData(100, "sata")]
+        [InlineData(101, "satayksi")]
+        [InlineData(345, "kolmesataaneljäkymmentäviisi")]
+        [InlineData(678, "kuusisataaseitsemänkymmentäkahdeksan")]
+        [InlineData(1000, "tuhat")]
+        [InlineData(1001, "tuhat yksi")]
+        [InlineData(1234, "tuhat kaksisataakolmekymmentäneljä")]
+        [InlineData(4567, "neljätuhatta viisisataakuusikymmentäseitsemän")]
+        [InlineData(10000, "kymmenentuhatta")]
+        [InlineData(100000, "satatuhatta")]
+        [InlineData(1000000, "miljoona")]
+        [InlineData(10000000, "kymmenenmiljoonaa")]
+        [InlineData(100000000, "satamiljoonaa")]
+        [InlineData(1000000000, "miljardi")]
+        [InlineData(2147483647, "kaksimiljardia sataneljäkymmentäseitsemänmiljoonaa neljäsataakahdeksankymmentäkolmetuhatta kuusisataaneljäkymmentäseitsemän")]  // int.MaxValue
```

Comment:
@MehdiK I actually got it right the first time, if you can believe that :grinning: